interfaces/Portalicious/src/app/domains/registration/registration.helper.ts

```diff
@@ -53,6 +54,18 @@ export const REGISTRATION_STATUS_VERB: Record<RegistrationStatusEnum, string> =
     [RegistrationStatusEnum.deleted]: $localize`Delete`,
   };
 
+export const LANGUAGE_ISO_639_1: Record<LanguageEnum, string> = {
+  ar: $localize`Arabic`,
+  en: $localize`English`,
+  es: $localize`Spanish`,
+  fr: $localize`French`,
+  in: $localize`Indonesian`,
+  nl: $localize`Dutch`,
+  pt_BR: $localize`Portuguese`,
```

Comment:
`<hat mode="pedantic">`
This one includes a 'region subtag', which is part of https://en.wikipedia.org/wiki/IETF_language_tag
`</hat>`

So the variable-name is a bit _too specific_; Wouldn't `LanguageName` or `LanguageLabel` work?

Reply:
I though it would be nice to have a name like this so if we ever add a new language, we don't just make something up and look a the standard. 

What do you think about using a specific const name? And what would be the right specific const name?

Reply:
+1 to what Elwin suggests. My two cents on the name itself: `LanguageEnumLabel`.

Reply:
If I put it in the 121-service it is angry about $localize. I there for just added your iso comment to https://github.com/global-121/121-platform/blob/main/services/121-service/src/shared/enum/language.enums.ts#L1 and change the name of the const to `LANGUAGE_ENUM_LABEL`